app/helpers/summary_helper.rb

```diff
@@ -12,10 +12,18 @@ def build_section(summary_text, object, fields)
 
   private
 
-  def build_header(summary_name)
+  def build_header(summary_name, link_title, link_url)
     content_tag(:div, class: 'row') do
       content_tag(:div, class: 'small-12 medium-7 large-8 columns') do
         content_tag(:h4, "#{summary_name}")
+      end + build_link(link_title, link_url)
+    end
+  end
+
+  def build_link(link_title, link_url)
+    if link_title.present? && link_url.present?
+      content_tag(:div, class: 'small-12 medium-4 large-3 columns') do
```

Comment:
I'd have set the classes to 'medium-5 large-4', that will balance with the foundation 12 columns 'medium-7 large-8' of the H4 row above